src/lib/database/seed.ts

```diff
@@ -34,6 +34,7 @@ export const seed = (db: Database): Seed => {
     name: "My Workspace",
     workspace_id: "seed_workspace_1",
     publishable_key: "seam_pk1_token",
+    is_sandbox: true,
```

Comment:
I think this makes sense, but if we don't want to make a potentially breaking chance for downstream consumers you can always update the seed in the repo that needs this.

What if another downstream consumer needs this to be non-,seed? Would this keep flip flopping here?

Reply:
> can always update the seed in the repo that needs this.
yeah that's what i'm currently doing, just made sense to me that we have 1 sandboxed and 1 normal workspace. 

Sorry what do you mean "flip flopping", this would just initialize the workspace to be sandboxed correct? Don't think there's logic that would change it during runtime

Reply:
this shouldn't change behavior in a major way so I think it's ok

Reply:
Ah I just mean if another consumer wanted this to be a non-sandbox workspace, then they might open a PR to change it back. But I think you're right that the workspace should be sandbox, since the real API always makes a sandbox workspace first anyway.

Reply:
There's also a second workspace in the seed that isn't a sandbox but provides similar devices. Not sure if the consistency will be an issue